integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/aggregation/SingleFieldAggregationTypeCheckingAndConversionIT.java

```diff
@@ -492,33 +492,11 @@ void withConverter_conversionDisabled(SupportedSingleFieldAggregationExpectation
 
 		AggregationScenario<?> scenario = expectations.simple();
 		testValidAggregationWithConverterSetting(
-				scenario, mainIndex.createScope(), fieldPath, ValueConvert.NO,
+				scenario, mainIndex.createScope(), fieldPath, ValueModel.INDEX,
 				dataSet
 		);
 	}
 
-	@ParameterizedTest(name = "{0}")
-	@MethodSource("params")
-	void withConverter_conversionParser(SupportedSingleFieldAggregationExpectations<F> expectations,
-			DataSet<F> dataSet) {
-		String fieldPath = mainIndex.binding().fieldModels.get( expectations.fieldType() ).relativeFieldName;
-		AggregationScenario<?> scenario = expectations.withFieldType(
-				TypeAssertionHelper.parser( expectations.fieldType() ) );
-		if ( "range".equalsIgnoreCase( expectations.aggregationName() ) ) {
-			testValidAggregationWithConverterSetting(
-					scenario, mainIndex.createScope(), fieldPath, ValueConvert.PARSE,
-					dataSet
-			);
-		}
-		else {
-			assertThatThrownBy( () -> scenario.setupWithConverterSetting( mainIndex.createScope().aggregation(), fieldPath,
-					ValueConvert.PARSE ) )
-					.isInstanceOf( SearchException.class )
-					.hasMessageContainingAll(
-							"Cannot use ValueConvert.PARSE as a converter. Use one of the allowed values [YES, NO] instead" );
-		}
-	}
-
```

Comment:
I get that this shouldn't fail anymore, but there's still behavior to be tested, isn't there?

Reply:
Thanks for taking a look! 🎉 
Yeah, we'd need to add more tests for `RAW` and `STRING` now that they should work everywhere 🙈
For now, I was just trying to make it work and see if what we'll get would be something we'll keep, as I wasn't convinced with a single enum 😄